<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddBookingDialog</class>
 <widget class="QDialog" name="AddBookingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>536</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_7">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_7">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string/>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QRadioButton" name="radioButtonFlight">
              <property name="text">
               <string>Flugbuchung</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioButtonCar">
              <property name="text">
               <string>Mietwagenreservierung</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioButtonHotel">
              <property name="text">
               <string>Hotelreservierung</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string/>
         </property>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>20</y>
            <width>147</width>
            <height>61</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QRadioButton" name="radioButtonNeueReise">
             <property name="text">
              <string>Neue Reise</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButtonBestehendeReise">
             <property name="text">
              <string>Bestehende Reise</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTabWidget" name="tabWidget_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>Tab 1</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label_21">
             <property name="text">
              <string>Reise-ID</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spinBoxReiseID"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>Tab 2</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_22">
             <property name="text">
              <string>Kunden-ID</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spinBoxKundenID"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <property name="documentMode">
        <bool>false</bool>
       </property>
       <widget class="QWidget" name="tabFlight">
        <attribute name="title">
         <string>Flugbuchung</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <layout class="QFormLayout" name="formLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Ankunftsdatum</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QDateEdit" name="dateEditAnkunft"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Abreisedatum</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QDateEdit" name="dateEditAbreise"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Preis</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QDoubleSpinBox" name="doubleSpinBoxPreis">
             <property name="maximum">
              <double>999999.000000000000000</double>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Abflugsort (iata-code)</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLineEdit" name="lineEditAbflugort"/>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Ankunftsort (iata-code)</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QLineEdit" name="lineEditAnkunftsort"/>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Airline</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QLineEdit" name="lineEditAirline"/>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Flugklasse (Y,W,J,F)</string>
             </property>
            </widget>
           </item>
           <item row="6" column="1">
            <widget class="QLineEdit" name="lineEditFlugklasse"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tabCar">
        <attribute name="title">
         <string>Mietwagenreservierung</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QFormLayout" name="formLayout_2">
           <item row="0" column="0">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Ankunftsdatum</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QDateEdit" name="dateEditAnkunft_2"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>Abreisedatum</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QDateEdit" name="dateEditAbreise_2"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>Preis</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QDoubleSpinBox" name="doubleSpinBoxPreis_2">
             <property name="maximum">
              <double>999999.000000000000000</double>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>Abholort</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLineEdit" name="lineEditAbholort"/>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>Rueckgabeort</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QLineEdit" name="lineEditRueckgabeort"/>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="label_13">
             <property name="text">
              <string>Firma</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QLineEdit" name="lineEditFirma"/>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="label_14">
             <property name="text">
              <string>Fahrzeugklasse</string>
             </property>
            </widget>
           </item>
           <item row="6" column="1">
            <widget class="QLineEdit" name="lineEditFahrzeugklasse"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tabHotel">
        <attribute name="title">
         <string>Hotelreservierung</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QFormLayout" name="formLayout_3">
           <item row="0" column="0">
            <widget class="QLabel" name="label_15">
             <property name="text">
              <string>Ankunftsdatum</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QDateEdit" name="dateEditAnkunft_3"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_16">
             <property name="text">
              <string>Abreisedatum</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QDateEdit" name="dateEditAbreise_3"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_17">
             <property name="text">
              <string>Preis</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QDoubleSpinBox" name="doubleSpinBoxPreis_3">
             <property name="maximum">
              <double>999999.000000000000000</double>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_18">
             <property name="text">
              <string>Hotel</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLineEdit" name="lineEditHotel"/>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_19">
             <property name="text">
              <string>Stadt</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QLineEdit" name="lineEditStadt"/>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="label_20">
             <property name="text">
              <string>Zimmer (EZ,DZ,AP,SU)</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QLineEdit" name="lineEditZimmer"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string/>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <layout class="QFormLayout" name="formLayout_4">
          <item row="0" column="0">
           <widget class="QLabel" name="label_23">
            <property name="text">
             <string>Predecessor 1</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="lineEditPre1"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_24">
            <property name="text">
             <string>Predecessor 2</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="lineEditPre2"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddBookingDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddBookingDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
